renovate.json5

```diff
@@ -31,5 +31,27 @@
       // difficult (or impossible) to implement so we just disable it entirely.
       automerge: false,
     },
+    // Angular major version updates attempt to upgrade fixtures for Angular 17, 18 etc
+    // we never want to upgrade those, so we disable them in fixtures.
+    // Angular major versions are being released on 6 months schedule and we will need to handle them
+    // manually anyway (even if it's just creating test fixtures for them, without adjusting the runtime
+    // itself)
+    // Additionally peerDeps bumps (like typescript) make it seemingly impossible to automate this process
+    // for demo app, so major bumps are just fully disabled.
+    {
+      description: 'Disable angular major version upgrades',
+      matchFileNames: ['tests/**/fixtures/**/package.json', 'demo/package.json'],
+      matchPackageNames: ['@angular/**', '@angular-devkit/**'],
+      matchUpdateTypes: ['major'],
+      enabled: false,
```

Comment:
Maybe we could just not set the `bump-framework-in-fixtures` label for these instead of disabling entirely? We wouldn't get the noisy failure notifications but we'd still have a little bit of visibility into available major bumps.

Reply:
If we could make that work, then it would be great!

Some troubles I did have with angular setup specifically is that "breaking changes" is mix of major and minor (0.X range for `zone.js`) which afaik is really painful to handle with renovate. Maybe there is some nicer setting than `updateType` (major/minor/patch) to play with. I'm not actually sure how to configure not allowing minor bumps for `zone.js` for non-breaking changes PRs while allowing this for breaking changes PRs. And then conditionally applying that label

In any case - I'll merge this as-is to get currently supported majors to receive correct non-breaking version bumps and we can look into making major version bumps as follow up